.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## PR Checklist
+Please check if your PR fulfills the following requirements:
+​
+- [ ] Changelog entry have been added / updated / is not needed
+- [ ] Docs/Stories have been added / updated (for bug fixes / features)
+​
+​
+## PR Type
+What kind of change does this PR introduce?
+​
+<!-- Please check the one that applies to this PR using "x". -->
```

Comment:
I think, it would be better to delete all types that do not apply. Otherwise the PR Type takes up a lot of space and provides little info for the reviewer. Maybe:

```suggestion
<!-- Delete all but the appropriate option(s) -->
```

I wrote option(s) because there may be cases where more than one applies (e.g. Build related and CI related changes)